Atomic/util.py

```diff
@@ -472,3 +486,11 @@ def validate_manifest(spec, img_rootfs=None, img_tar=None, keywords=""):
     if keywords:
         cmd += ['-k',keywords]
     return subp(cmd)
+
+def get_signature_path(atomic_conf):
+    # If signature path is defined, get it; else return default
+    signature_path = get_atomic_config_item(['default-sigstore-path'], atomic_config=atomic_conf)
+    if signature_path is None:
+        signature_path = '/etc/pki/containers/'
+    return signature_path
```

Comment:
We need to make this consistent with https://github.com/mtrmac/image/blob/docker-lookaside/docker/lookaside.go#L102 : 

 - Using URLs
 - `"registries".""` instead of `default-sigstore-path` (though that might be by changing `containers/image` instead)
 - `sigstore-write` values
 - No `/etc/pki/containers` default I think… A single sigstore for all of the internet really isn’t likely to be useful. For any external public repositories, there would be a read-only sigstore via HTTP. Read-write would be only the locally created/managed repositories (`mydepartment.mycompany.com/registry`) or something.